packages/hop-node/src/watchers/CommitTransferWatcher.ts

```diff
@@ -177,11 +178,12 @@ class CommitTransfersWatcher extends BaseWatcherWithEventHandlers {
       }
 
       // Define new object on first run after server restart
-      if (!this.isCommitTxSent[destinationChainId]) {
-        this.isCommitTxSent[destinationChainId] = false
-      }
-
-      if (this.isCommitTxSent[destinationChainId]) {
+      const timestampOk = this.commitTxSentAt[destinationChainId] + TX_RETRY_DELAY_MS < Date.now()
```

Comment:
by default `this.commitTxSentAt[destinationChainId]` will be undefined so it'll evaluate to NaN < Date.now(). I don't think it affects anything here but an fyi

Reply:
Sounds good. We do this elsewhere and have tested it before, so I believe we are good to go here.